androguard/core/axml/__init__.py

```diff
@@ -1029,6 +1040,16 @@ def __init__(self, raw_buff: bytes)->bytes:
                     logger.warning("Not all namespace mappings were closed! Malformed AXML?")
                 break
 
+    def clean_and_replace_nsmap(self, nsmap, invalid_prefix):
+        correct_prefix = 'android'
+        corrected_nsmap = {}
+        for prefix, uri in nsmap.items():
+            if prefix.startswith(invalid_prefix):
```

Comment:
Why `.startswith()`? AFAICT the error should return the invalid prefix. Not a prefix of the invalid prefix :)